frontend/src/toolbar/stats/CurrentPage.js

```diff
@@ -48,9 +49,9 @@ export function CurrentPage() {
                         />
                     ) : (
                         <GlobalOutlined />
```

Comment:
For some reason this is now poorly aligned: 
![image](https://user-images.githubusercontent.com/1727427/85543116-0f47f380-b61a-11ea-9a36-34f91f222413.png)

Reply:
Fixed!
![image](https://user-images.githubusercontent.com/53387/85552629-7918cb00-b623-11ea-863e-608bf72289bc.png)
![image](https://user-images.githubusercontent.com/53387/85552661-833ac980-b623-11ea-87ac-6d2e2a5b5d25.png)